e2e/test/scenarios/dashboard-filters/reproductions/24235-exlude-all-date-options.cy.spec.js

```diff
@@ -43,11 +43,15 @@ describe("issue 24235", () => {
     );
 
     filterWidget().contains(parameter.name).click();
-
     popover().within(() => {
       cy.findByText("Exclude...").click();
       cy.findByText("Days of the week...").click();
       cy.findByText("Select none...").click();
+      cy.findByText("Add filter").click();
+    });
+
+    filterWidget().click();
+    popover().within(() => {
```

Comment:
Not sure how this tests was supposed to work, but clicking "Select none" without actually applying the filter didn't do anything. So I split this into two actions to verify it properly.